### PR TITLE
rhcos: Bump to 410.8.20190508.1

### PR DESCRIPTION
**Pull request by ashcrow**
/cc @tbielawa @miabbott @yuqi-zhang @darkmuggle @cgwalters @abhinavdahiya

### CODE REVIEW

**Comment by ashcrow** [comment]
@yuqi-zhang updated.

*Note*: We have confirmation from @tbielawa that this is :+1: version.

File: data/data/rhcos.json [hunk 1]
```diff
@@ -1,104 +1,111 @@
...
-            "hvm": "ami-02b30dab0946cf89d"
+            "hvm": "ami-096cb19297a5c2726"
         }
     },
-    "baseURI": "https://releases-art-rhcos.svc.ci.openshift.org/art/storage/releases/ootpa/410.8.20190425.0/",
-    "buildid": "410.8.20190425.0",
+    "baseURI": "https://releases-art-rhcos.svc.ci.openshift.org/art/storage/releases/ootpa/410.8.20190508.1/",
+    "buildid": "410.8.20190508.1",
     "images": {
+        "aws": {
```
Comment (yuqi-zhang) [inline data/data/rhcos.json]: Hmm I feel like we removed this from the artifacts at some point, not sure how it got added back in, either case shouldn't affect anything

Comment (ashcrow) [reply data/data/rhcos.json]: I'll remove it.

Comment (ashcrow) [reply data/data/rhcos.json]: Removed

**Comment by ashcrow** [comment]
```
../../coreos/terraform-provider-matchbox/main.go:11:15: cannot use matchbox.Provider (type func() "github.com/poseidon/terraform-provider-matchbox/vendor/github.com/hashicorp/terraform/terraform".ResourceProvider) as type "github.com/coreos/terraform-provider-matchbox/vendor/github.com/hashicorp/terraform/plugin".ProviderFunc in field value
```

**Comment by ashcrow** [comment]
Failure:
```
../../coreos/terraform-provider-matchbox/main.go:11:15: cannot use matchbox.Provider (type func() "github.com/poseidon/terraform-provider-matchbox/vendor/github.com/hashicorp/terraform/terraform".ResourceProvider) as type "github.com/coreos/terraform-provider-matchbox/vendor/github.com/hashicorp/terraform/plugin".ProviderFunc in field value
```

**Comment by ashcrow** [comment]
Failures are unrelated to this PR.

**Comment by ashcrow** [comment]
https://github.com/openshift/installer/pull/1730 is the fix for the e2e failures we are seeing

**Comment by ashcrow** [comment]
Upstream issue fix merged.

/retest

**Comment by ashcrow** [comment]
Another OpenStack flake :slightly_frowning_face: 

/test e2e-openstack

**Comment by abhinavdahiya** [comment]
> Another OpenStack flake slightly_frowning_face
> 
> /test e2e-openstack

openstack PR is not blocking. Will let openstack team know.

**Comment by ashcrow** [comment]
/test e2e-openstack

Come on OpenStack!

**Comment by abhinavdahiya** [comment]
/hold

until https://github.com/openshift/installer/pull/1727 merges _that's a beta5 blocker_

**Comment by abhinavdahiya** [comment]
/hold cancel
https://github.com/openshift/installer/pull/1727 merged

**Comment by ashcrow** [comment]
/test e2e-openstack

Same flake

```
 One of the configured repositories failed (Unknown),
 and yum doesn't have enough cached data to continue. At this point the only
 safe thing yum can do is fail. There are a few ways to work "fix" this:
```

**Comment by ashcrow** [comment]
/test e2e-openstack

:confounded:

**Comment by cgwalters** [comment]
It doesn't look like a flake but a configuration change; see https://github.com/openshift/installer/pull/1733

I think either an admin needs to override the openstack context, or the installer merges that PR (and have it work).

**Comment by abhinavdahiya** [comment]
> It doesn't look like a flake but a configuration change; see #1733
> 
> I think either an admin needs to override the openstack context, or the installer merges that PR (and have it work).

like I said before, openstack is known failing currently and is not blocking for merge _for now_
https://github.com/openshift/installer/pull/1731#issuecomment-490600621

**Comment by cgwalters** [comment]
Like this but I doubt I have perms:
/override e2e-openstack

**Pull request merged**